plugins/protoc-gen-json.py

```diff
@@ -103,50 +103,50 @@ def parse_field(f):
         field["type_lua"] = "number"
         field["type_cpp"] = "double"
         field["type_cpp_lower"] = "double"
-        field["type_is_allocated"] = False
+        field["type_is_allocated"] = field["repeated"]
```

Comment:
we `malloc` repeated values https://github.com/defold/extension-proto/blob/fe5c4500791a21550895de2f7c78061e4c3aed74/extension-proto.cpp.mtl#L201

so, we need to make sure we call `free` for them